test-utils/src/e2e_test_env.rs

```diff
@@ -361,7 +363,18 @@ where
         } else {
             StateMerkleTreeConfig::default()
         };
-        println!("config: {:?}", config);
+        println!("merkle tree config: {:?}", merkle_tree_config);
+        let queue_config = if !self.keypair_action_config.fee_assert {
+            let capacity: f64 = gen_prime(&mut self.rng, 1.0..10000.0).unwrap();
```

Comment:
it should work with any number primes are just about efficiency right?

Reply:
Without prime numbers there is a higher chance of collisions and quadratic probe not doing the job properly, therefore the queue might fill up very quick

Reply:
this test is just about ensuring that the account layout works for any input though not about efficiency